src/dotnet/UI.Blazor.App/Pages/Test/EmbeddedTestPage.razor

```diff
@@ -0,0 +1,45 @@
+@page "/test/embedded"
+@using ActualChat.Hosting
+
+@{
+    var baseUri = new Uri(HostInfo.BaseUrl);
+    var domain = HostInfo.IsDevelopmentInstance
```

Comment:
Can you explain with comments what this trick with "cdn.local.actual.chat" domain means?